test/external-modules/delayed-aggs/build.gradle

```diff
@@ -23,3 +19,8 @@ restResources {
     include '_common', 'indices', 'index', 'cluster', 'search'
   }
 }
+
+tasks.named('yamlRestTest') {
+  def isSnapshot = buildParams.snapshotBuild
```

Comment:
@breskeby I had to do this in a number of places, otherwise the configuration cache would throw an error

Reply:
Yeah. Onlyif is evaluated at tasks execution phase and buildparsms here is an extension. Those can not be accessed during execution phase